### ISSUE TITLE
App won't start on Sony RX 100 IV

### ISSUE THREAD

**Issue by haoliu119** [Bostwickenator/STGUploader#11]
Thank you for making this app @Bostwickenator 
I successfully installed the STG uploader app via the Safari browser plugin method and verified that `org.bostwickenator.googlephotos (1.4` is installed on my Sony RX 100 IV camera.

I start my camera and navigate to the application list, I see the STG app. However every time I start the app on my camera, all it does is restarting my camera, and nothing else happens.

I wonder if anyone else has run into this issue. Thanks.

**Comment by tzwalf** [comment]
Yes, me too. It's also refusing to upload photos. Both are intermittent. I've taken to uninstalling/reinstalling to work around the issue, but it always comes back after a few upload sessions.

**Comment by haoliu119** [comment]
I found a workaround that seems to work every time. I need to launch the playmemories apps store from the application list and wait for it to load first. Then I exit the apps store and launch the STG uploader app, and it works. Hope this information helps with future improvements.

**Comment by Bostwickenator** [comment]
Thanks for the reports. Are you still having these issues?

**Comment by tzwalf** [comment]
Hi Alex, yes, still having these issues. The workaround above works, but eventually I run in to another issue, which is that it will say upload complete, but there are still photos remaining. To work around that one I uninstall/reinstall. There is another thread about that one so I'll comment there.
Thanks so much for your work on this application - it makes our camera so good to use.
